usagov/vote-gov
# [Vote 646] Spanish link test along with updates

link to ticket -> https://bixal-projects.atlassian.net/browse/VOTE-646

This ticket will add a few things 
1- a test for Spanish pages for external link checking 
2- set up a cron job to run once a week (currently it is set for 9am on Thursday for testing but will up to 9am on Monday once we confirm that it is working 

also added the internal link testing with the sitemap to give up more robust testing for that coverage 

to test
-pull down branch 
- run `npm run start`
- spilt terminal 
- run `npm run cy:open` will open the browser or `npm run cy:proofer` this will run the external links headless 
- check that external links is working as expected
